evm/src/main/java/org/hyperledger/besu/evm/operation/CreateOperation.java

```diff
@@ -21,8 +21,8 @@
 
 public class CreateOperation extends AbstractCreateOperation {
 
-  public CreateOperation(final GasCalculator gasCalculator) {
-    super(0xF0, "CREATE", 3, 1, gasCalculator);
+  public CreateOperation(final GasCalculator gasCalculator, final int maxInitcodeSize) {
```

Comment:
Can you explain why we need the check at the EVM level as well as transaction validation level?

Reply:
Transaction validation is top level and checked before any EVM operations are executed.  Once we are in the operations we have accepted the transaction as valid and are executing it. These are two distinct different cases.  The EIP also recognizes this as rules 1&3 cover validation and recognize the different contexts.